main.py

```diff
@@ -136,6 +141,10 @@ def BoardInit(args):
             board.config_board('p52')
             heart_window_seconds = biometrics_logic.window_seconds
             startup_time = max(startup_time, heart_window_seconds)
+        
+        ### Add ml action to logics if enabled
+        if args.enable_action:
+            logics.append(MLAction(board, ema_decay=ema_decay * 5)) # sped up for good feels

```

Comment:
I think it might be better to offer a secondary option to configure the decay value for this if we are applying an internal calculation. It should be ok to keep this by default but we shouldn't rely solely on hard-coded formulas if this may be useful for users to configure directly

Reply:
Will add soon :tm: